contrib/completion/zsh/_docker

```diff
@@ -599,6 +599,7 @@ _docker() {
         "($help)--default-gateway-v6[Container default gateway IPv6 address]:IPv6 address: " \
         "($help)*--dns=-[DNS server to use]:DNS: " \
         "($help)*--dns-search=-[DNS search domains to use]" \
+        "($help)*--dns-opt=-[Set custom DNS options]" \
```

Comment:
Can you modify this to `"($help)*--dns-opt=-[Set custom DNS options]:DNS option: " \`. This adds a help message to the user.
You can also a modify the `--dns-search` to `"($help)*--dns-search=-[DNS search domains to use]:DNS search: " \` for the same reason.

Also, `--dns-opt` need to be added between line 230 and 231 for the `create` and `run` command.